cmd/juju/commands/bootstrap.go

```diff
@@ -564,45 +568,69 @@ func (c *bootstrapCommand) Run(ctx *cmd.Context) (resultErr error) {
 		return errors.Trace(err)
 	}
 
-	hostedModelUUID, err := utils.NewUUID()
-	if err != nil {
-		return errors.Trace(err)
-	}
+	getHostedModel := func() (*jujuclient.ModelDetails, error) {
```

Comment:
This is large enough that it should be a non-inline func

Reply:
agreed, moved to a separate method